config/settings.yml

```diff
@@ -1298,6 +1298,10 @@ vanotify:
         claim_submission_error_text: fake_template_id
     benefits_decision_review:
       api_key: fake_secret
+    benefits_disability:
+      api_key: fake_secret
+      template_id:
+        form526_document_upload_failure_notification_template_id: form526_document_upload_failure_notification_template_id
```

Comment:
Note I have a ticket linked in the description to set up these values in production